Sources/PubNubChatComponents/Message List/MessageListComponentViewModel.swift

```diff
@@ -164,7 +165,7 @@ open class MessageListComponentViewModel<ModelData, ManagedEntities>:
       collectionViewType: messageListTheme.collectionViewTheme.viewType,
       collectionViewLayout: layout,
       messageInputComponent: messageInputComponent,
-      enableReactions: messageListTheme.enableReactions
+      reactionTheme: messageListTheme.reactionTheme
```

Comment:
I removed enableReactions in favour of reactionTheme, my intention was to check if theme exists, if it is there then we can show picker and reactions.